frontend/app/src/utils/balances.ts

```diff
@@ -203,9 +203,16 @@ export const accountsWithBalances = (
   blockchain: Exclude<Blockchain, 'BTC'>
 ): AccountWithBalance[] => {
   const data: AccountWithBalance[] = [];
+  const { getChainInfoById } = useSupportedChains();
+
+  let nativeAsset: string = blockchain;
+  const chainInfo = get(getChainInfoById(blockchain));
+  if (chainInfo && 'nativeAsset' in chainInfo) {
+    nativeAsset = chainInfo.nativeAsset;
+  }
```

Comment:
So native asset is only on evm chains and it is only if the asset is different from the id, maybe we could move it to useSupportedChains and use the evmChainData?